engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/gateway/InclusiveGatewayTest.java

```diff
@@ -74,7 +77,7 @@ public void shouldSplitOnInclusiveGateway() {
             .moveToLastInclusiveGateway()
             .defaultFlow()
             .sequenceFlowId("s3")
-            .conditionExpression("= contains(str,\"c\")")
+            .conditionExpression("= missing_var")
```

Comment:
❌ This condition is distracting a lot from the intent of this test case.

🔧 Let's make it more clear that the default condition is ignored here by changing the default flow's condition to `= true`